reservation_units/tests/test_reservation_unit_export.py

```diff
@@ -0,0 +1,130 @@
+import csv
+import shutil
+from pathlib import Path
+from typing import Any, List
+
+from assertpy import assert_that
+from django.conf import settings
+from django.test.testcases import TestCase
+from django.utils import timezone
+
+from ..utils.export_data import ReservationUnitExporter
+from .factories import ReservationUnitFactory
+
+
+class ReservationUnitDataExporterTestCase(TestCase):
+    export_dir = Path(settings.BASE_DIR) / "exports" / "reservation_unit_exports"
+    file_name = f"reservation_units__{timezone.now().strftime('%d-%m-%Y')}.csv"
```

Comment:
now my eyes are starting to see these double underscores :eyes:

Reply:
Yeah, this is one of those "up to preference" things :smile: .

I used double underscores in the filename to specifically differentiate between name and date. There are many naming conventions, that could be used.

```
reservation_units__10-3-2022.csv
reservation-units_10-3-2022.csv
"reservation-units 10-3-2022.csv"
ReservationUnits_10-3-2022.csv
etc...
```

What would you say, is the one I chose good or should these be changed?

Reply:
This is a taste thing and nothing to be changed - I would just said the file to be `reservation_units_10-3-2022.csv` 
No need to change, the one you chose is good :heavy_check_mark:  =>  lets merge :shipit: